socs/agents/acu/agent.py

```diff
@@ -1076,6 +1076,10 @@ def go_to(self, session, params):
             if not acquired:
                 return False, f"Operation failed: {self.azel_lock.job} is running."
 
+            self.log.info('Clearing faults to prepare for motion.')
+            yield self.acu_control.clear_faults()
+            yield dsleep(1)
+
```

Comment:
Should we have a similar block before boresight motion?

Reply:
yes, yes we should ...